src/partials/section-gallery.html

```diff
@@ -0,0 +1,98 @@
+<div class="section-gallery">
+  <div class="container">
+    <div class="left-section">
+      <div class="panel-bar">
+        <div class="row no-gutters align-items-end">
+          <div class="col-auto heading">
+            <h3>Furniture Gallery</h3>
+          </div>
+        </div>
+      </div>
+      <div class="container-menu-content-sllider">
+        <div class="menu-gallery">
+          <ul>
+            <li>Featured</li>
+            <span></span>
+            <li>Top Seller</li>
+            <span></span>
+            <li>Sale OFF</li>
+            <span></span>
+            <li>Top Rated</li>
+          </ul>
+        </div>
+        <div class="content-gallery">
+          <div class="outlines">
+            <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
+            <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+            <a href="#" class="btn-outline"><i class="far fa-eye"></i></a>
+            <a href="#" class="btn-outline"
```

Comment:
Witam. Fajne wykonanie 👍 . Jedynie co mógłbyś zmienić to dodać małe odstępy (margin) między tagami <a> (fragment gdzie jest serce, koszyk itp.) na stronie stykają się ze sobą.

Reply:
dzięki. Poprawiłem